packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -1,30 +1,149 @@
 # Dropdown
 
+## Basic usage
+
 ```html
-<div class="Dropdown is-open">
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#info" />
-    </svg>
-    <span>Information</span>
-  </a>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#link" />
-    </svg>
-    <span>More links</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#profile" />
-    </svg>
-    <span>Profile</span>
-  </a>
-  <a href="#" class="d-flex">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#help" />
-    </svg>
-    <span>Help</span>
+<div class="DropdownWrapper">
+  <button
```

Comment:
Maybe `type` attribute is missing here. (And all other `button` elements within this PR.)